src/left-nav.jsx

```diff
@@ -225,6 +236,28 @@ const LeftNav = React.createClass({
       </div>
     );
   },
+  
+  _isOpen() {
```

Comment:
Well there are 2 sources of **truth** about whether the left nav is open. `this.state.open` and `this.props.open`, I'm afraid if we keep the state up-to-date with props it will be more complex. But if you have something on your mind, please let me know so I can fix it somehow :grin: